schemas/context/opportunitycontactrole.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunitycontactrole",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Opportunity Contact Role",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "Opportunity Contact Role",
+  "definitions": {
+    "opportunityContactRole": {
+      "properties": {
+        "xdm:opportunityContactID": {
+          "title": "opportunity ContactID",
+          "description": "Technical PK for Opportunity Contact.",
+          "type": "string"
+        },
+        "xdm:opportunityID": {
+          "title": "Opportunity ID",
+          "description": "Opportunity ID unique identifier.",
+          "type": "string"
+        },
+        "xdm:contactID": {
+          "title": "Contact ID",
+          "description": "Contact ID unique identifier.",
+          "type": "string"
+        },
+        "xdm:roleID": {
+          "title": "Contact Role ID",
+          "description": "Contact role unique identifier.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/opportunityContactRole"
+    }
+  ],
```

Comment:
Please add "$ref": "https://ns.adobe.com/xdm/data/record" to allOf as its in the meta:extends.